src/diffusers/pipelines/ddim/pipeline_ddim.py

```diff
@@ -29,20 +29,16 @@ def __init__(self, unet, scheduler):
 
     @torch.no_grad()
     def __call__(
-        self, batch_size=1, generator=None, torch_device=None, eta=0.0, num_inference_steps=50, output_type="pil"
+        self, batch_size=1, generator=None, eta=0.0, num_inference_steps=50, output_type="pil"
```

Comment:
`kwargs` here would help with backwards compatibility after we remove `torch_device`

Reply:
yes pedro could we do something similar to how we do it in `transformers`? 
E.g.:
```suggestion
        self, batch_size=1, generator=None, eta=0.0, num_inference_steps=50, output_type="pil", **kwargs
        ):
        
        if "torch_device" in kwargs:
            device = kwargs.pop("torch_device")
            warnings.warn(
                "`torch_device` is deprecated as an input argument to `__call__` and will be removed in v0.3.0.
                Consider using `pipe.to(torch_device)` instead."
           )
        # ...set device as previously 
```

Reply:
And if `torch_device` is passed we still use it to overwrite (but we can remove it in v0.3.0 already :-))

Reply:
If possible we can apply this to all pipelines